aspnetcore/migration/22-to-30.md

```diff
@@ -340,14 +340,14 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
         });
 ```
 
-If the app creates the host manually with `HostBuilder`, call `UseKestrel` on the web host builder in `ConfigureWebHostDefaults`:
+If the app creates the host manually with `ConfigureWebHost` instead `ConfigureWebHostDefaults`, call `UseKestrel` on the web host builder:
```

Comment:
```suggestion
If the app creates the host manually with `ConfigureWebHost` instead of `ConfigureWebHostDefaults`, call `UseKestrel` on the web host builder:
```